quodlibet/qltk/quodlibetwindow.py

```diff
@@ -990,7 +990,7 @@ def check_updates_handler(*args):
             name="RefreshLibrary", label=_("_Scan Library"),
             icon_name=Icons.VIEW_REFRESH)
         act.connect('activate', self.__rebuild, False)
-        ag.add_action(act)
+        ag.add_action_with_accel(act, "<Primary>R")
```

Comment:
If we want this, we should add it to documentation too I'd say

Reply:
The feature is self-documenting because Gtk will show the keyboard shortcut in the menu. We're not explicitly documenting Ctrl+B for editing bookmarks.